OasisDEX/oasis-borrow
# Gnosis Safe Dirty Fix

# [Ticket in Shortcut](https://app.shortcut.com/oazo-apps/story/6134/multiple-vault-gas-estimates-failing-w-gnosis-safe-and-not-sending-t-x)
  
## Changes 👷‍♀️
- Skip gas estimation for specific multiply vault actions with gnosis safe
  
## How to test 🧪
- Create Gnosis Safe
- Go to gnosis safe on mainnet or goerli
- Open an ETH-A multiply vault or manage existing
- Withdraw some ETH 
- Also try and close vault

Expect gas estimation to fail (n/a) but for transaction to be sent correctly by gnosis provider. Should see popup modal and correct gas estimation in there.
